Guide.docc/IncrementalAdoption.md

```diff
@@ -149,6 +149,60 @@ class UIStyler {
 Combining static and dynamic isolation can be a powerful tool to keep the
 scope of changes gradual.
 
+## Missing Annotations
+
+Dynamic isolation gives you tools to express isolation at runtime.
+But, you may also find you need to describe other concurrency properties
+that are missing from unmigrated modules.
+
+### Unmarked Sendable Closures
+
+A callback that is missing a `Sendable` annotation can be a problem when
+combined with actor-isolated types.
```

Comment:
We should intro this a bit more, maybe...

```suggestion
Closures can be @Sendable or not, and this affects how isolation is inferred for them when they are used in an isolated context (e.g. in an actor, or function with an `isolated` parameter).

For example a callback that is intended to be called on "some arbitrary thread",
that is missing a `Sendable` annotation can be a problem when combined with actors.
```

Reply:
yeah you are right. Let me think on this just a little more.

Reply:
I've adjusted the wording here quite.